src/System.Net.Utilities/src/System/Net/NetworkInformation/Icmpv4MessageConstants.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    internal static class Icmpv4MessageConstants
+    {
+        // Message Types
+        internal const int EchoReply = 0;
```

Comment:
I actually did originally have them as enums, but it was really verbose in the big switch statement below, so I switched it over to this. I agree that it technically should be an enum so I will try switching it back.